### PR TITLE
Yda 5393 copy to vault

### PR DESCRIPTION
**Pull request by claravox**
requires https://github.com/UtrechtUniversity/yoda/pull/427

### CODE REVIEW

**Comment by claravox** [comment]
API tests pass locally. May have to do with the new configurations added

File: vault.py [hunk 1]
```diff
@@ -828,24 +829,60 @@ def api_revoke_read_access_research_group(ctx, coll):
...
+    :returns: True for successful copy
     """
-    destination = target + '/original'
-    origin = folder
+    returncode = 0
+    try:
+        returncode = subprocess.call(["irsync", "-rK", "i:{}/".format(coll), "i:{}/original".format(target)])
+    except Exception as e:
+        log.write(ctx, "irsync failure: " + e)
+        log.write(ctx, "irsync failure for coll <{}> and target <{}>".format(coll, target))
```
Comment (claravox) [inline vault.py:909]: I have never seen these lines run, even when clearly something went wrong because of wrong permissions on the folders.

Comment (claravox) [reply vault.py:909]: This is expected, as an exception would only occur when something wrong went with subprocess itself.

**Review by lwesterhof — APPROVED** [review]
LGTM!

**Pull request merged**